tv.kodi.Kodi.appData.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>tv.kodi.Kodi</id>
+    <name>Kodi</name>
+    <project_license>GPL-2.0-only GPL-2.0-or-later LGPL-2.1-or-later MIT BSD-3-Clause BSD-4-Clause</project_license>
+    <developer_name>Team Kodi</developer_name>
+    <summary>Media center</summary>
+    <url type="homepage">https://kodi.tv/</url>
```

Comment:
Add donation option https://kodi.tv/contribute/donate

Reply:
will do, also will add bugtracker, faq, help and translate